public/v4/apps/squidex.yml

```diff
@@ -0,0 +1,114 @@
+captainVersion: 4
+services:
+    $$cap_appname:
+        depends_on:
+            - $$cap_appname-mongo
+        image: 'squidex/squidex:$$cap_squidex_version'
+        volumes:
+            - '$$cap_appname-data:/app/Assets'
+        restart: always
+        environment:
+            URLS__BASEURL: 'https://$$cap_url'
+            URLS__ENFORCEHTTPS: 'True'
+            EVENTSTORE__MONGODB__CONFIGURATION: 'mongodb://root:$$cap_mongo_password@srv-captain--$$cap_appname-mongo'
+            STORE__MONGODB__CONFIGURATION: 'mongodb://root:$$cap_mongo_password@srv-captain--$$cap_appname-mongo'
+            IDENTITY__ADMINEMAIL: $$cap_admin_email
+            IDENTITY__ADMINPASSWORD: $$cap_admin_password
+            IDENTITY__GOOGLECLIENT: $$cap_google_client
+            IDENTITY__GOOGLESECRET: $$cap_google_secret
+            IDENTITY__GITHUBCLIENT: $$cap_github_client
+            IDENTITY__GITHUBSECRET: $$cap_github_secret
+            IDENTITY__MICROSOFTCLIENT: $$cap_microsoft_client
+            IDENTITY__MICROSOFTSECRET: $$cap_microsoft_secret
+            LETSENCRYPT_HOST: $$cap_url
+            LETSENCRYPT_EMAIL: $$cap_admin_email
+            VIRTUAL_HOST: $$cap_url
+        caproverExtra:
+            containerHttpPort: '80'
+    $$cap_appname-mongo:
+        image: 'mongo:$$cap_mongo_version'
+        volumes:
+            - '$$cap_appname-mongo-data:/data/db'
+            - '$$cap_appname-mongo-config:/data/configdb'
+        restart: always
+        environment:
+            MONGO_INITDB_DATABASE: squidex
+            MONGO_INITDB_ROOT_USERNAME: root
+            MONGO_INITDB_ROOT_PASSWORD: $$cap_mongo_password
+        caproverExtra:
+            notExposeAsWebApp: 'true'
+caproverOneClickApp:
+    variables:
+        - id: $$cap_squidex_version
+          label: Squidex Version
+          defaultValue: 4.6.0
+          description: >-
+              Check out their page for the valid versions
+              https://hub.docker.com/r/squidex/squidex/tags
+          validRegex: '/^([^\s^\/])+$/'
+        - id: $$cap_mongo_version
+          label: MongoDB Version
+          defaultValue: '4'
+          description: >-
+              Check out their Docker page for the valid tags
+              https://hub.docker.com/r/library/mongo/tags/
+          validRegex: '/^([a-zA-Z0-9])+$/'
+        - id: $$cap_mongo_password
+          label: MongoDB password
+          description: Only use alphanumeric chars.
+          validRegex: '/^([a-zA-Z0-9])+$/'
+        - id: $$cap_url
+          label: Squidex URL
+          defaultValue: squidex.example.com
```

Comment:
You don't need this variable, simply replace all instances of `$$cap_url` with `$$cap_appname.$$cap_root_domain`

Reply:
Replaced, thanks.
Also added a clarification that HTTPS is needed(otherwise login won't work). Probably some checks in their app.
+ changed appname-mongo to appname-db as I saw a lot of one-click-apps use this convention.